src/main/app/src/app/shared/material-form-builder/form-components/file-input/file-input.component.ts

```diff
@@ -179,6 +179,12 @@ export class FileInputComponent extends FormComponent implements OnInit {
       )}MB)`;
       return false;
     }
+
+    if (!file.size) {
+      this.data.uploadError = "Het bestand is leeg";
```

Comment:
Do we have tests for file (max, min/no-)size issues?

Reply:
Unfortunately, no. I'm going to try to consolidate the different file inputs later. When I do that, I'll make sure I'll add a unit test as well.